### DISCUSSION TITLE
AP LEDs not controllable

### DISCUSSION THREAD

**Discussion by danbenjamin23** [Art-of-WiFi/UniFi-API-client#204] [General]
Hello, it appears as though the site_leds($enable) function does not control the LED of Unifi APs. Is there currently a function for switching AP LEDs on or off? 

Note: I tried using the locate_ap function with my AP, and this works properly (LED flashes).

**Comment by malle-pietje** [comment]
We've noticed the site-wide LED control has disappeared from the web-interface in the later 7.X versions. Which controller version are you using and is there an option in the web-interface to control the LEDs site-wide?

**Reply by danbenjamin23** [reply]
I'm using Version 8.0.7 running locally on a Mac. There is no option to control LEDs on the site level, just on the device level.

**Comment by malle-pietje** [comment]
Have you tried whether the `led_override()` method works on a single device?

**Reply by danbenjamin23** [reply]
This worked! Thanks for your help.
